lib/helpers/url_validator.dart

```diff
@@ -2,7 +2,7 @@
 bool validateUrl(String? value) {
   String pattern =
       r'(^https://[a-zA-z/[.com,.edu,.gov,.org,.net]?]*)'; // NEEDS UPDATING LATER
-  RegExp regExp = new RegExp(pattern);
```

Comment:
why is it that the new foes nothing?

Reply:
that's a misleading diff. You had new in the declaration and instantiation, I removed one of them